jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -0,0 +1,210 @@
+import groovy.transform.Field;
```

Comment:
Per @PengTian0 , this is almost just change the file name and refine the code layout.
so assuming it functions the same as original code